vertical-pod-autoscaler/templates/admission-controller-service.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.admissionController.enabled }}
+{{- if .Values.admissionController.servicemonitor.enabled }}
```

Comment:
Shouldn't this be toggling only the `metrics` port? Atm the service won't be created if the service monitor is disabled, which I think it's not the intention, no?

Reply:
this is a typo actually. Shouldn't be set here